src/components/CallHistoryLastUpdatedStatus.tsx

```diff
@@ -18,14 +18,24 @@ const CallHistoryLastUpdatedStatus = (): ReactElement => {
         });
     }, []);
 
+    const DisplayResult = () => {
+        const date = mirBstDateFormatter(reportStatus, true);
+        if (date == "Invalid Date") {
+            return date;
+        }
+        return ` (${date})`;
```

Comment:
The fact that this has a space in front of it but the `Invalid Date` string doesn't really bothers me. I suspect there might be a formatting reason but I'd argue it's not the responsibility of `DisplayResult`; perhaps the space should go where it's called instead?